templates/etc/systemd/system/tmp.mount.j2

```diff
@@ -1,4 +1,4 @@
-## This file is managed by Ansible, YOUR CHANGED WILL BE LOST!
```

Comment:
Why not use conventional `{{ ansible_managed | comment }}`?

Reply:
Fixed
I didn't know this one, thanks!